pages/data-providers/[data-provider-id]/overview.jsx

```diff
@@ -129,6 +84,17 @@ const DepositingCard = ({ chartData, complianceLevel }) => {
   )
 }
 
+const DOICard = ({ value }) => (
+  <Card>
+    <h2>DOI coverage</h2>
+    <PerformanceChart value={value} increase={2} caption="DOI" />
+    {2 && <p>We can enrich you DOI coverage by {2}%.</p>}
```

Comment:
Maybe it's worth putting this into texts?

Reply:
Yes, I will do so